openlibrary/plugins/openlibrary/js/lists/ListService.js

```diff
@@ -42,6 +42,24 @@ export function createList(userKey, data, success) {
     });
 }
 
+/**
+ * Submits request to create new list.  Returns Promise.
+ *
+ * @param {string} userKey The patron's key, in the form "/people/{username}"
+ * @param {object} data Object containing the new list's name, description, and seeds.
+ * @returns {Promise<Response>} The results of the POST request
+ */
+export async function createList(userKey, data) {
```

Comment:
What happens if there is a request error/failure?

Reply:
Nothing now, but maybe we trigger a toast message?

Reply:
Quick note: any errors are meant to be handled by the caller of this function.  Whatever we decide to do on list creation failure will be included inside of a `catch()` call, here: https://github.com/internetarchive/openlibrary/blob/965c75f67bae55fb152ef6a9d2964119d0c5f6e9/openlibrary/plugins/openlibrary/js/my-books/ReadingLists.js#L379-L383